gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/pdeps/ResolveProductDependenciesTask.java

```diff
@@ -73,15 +72,30 @@
     abstract ConfigurableFileCollection getProductDependenciesFiles();
 
     @OutputFile
-    abstract RegularFileProperty getManifestFile();
+    public abstract RegularFileProperty getManifestFile();
```

Comment:
Need the getProductDependenciesFiles() property to also be public so can override the generation mechanism.